### DISCUSSION TITLE
Hello, may I ask how the connecting line can connect two notes together? The connecting line on the score is too short now,

### DISCUSSION THREAD

**Discussion by ruir666** [CoderLine/alphaTab#1138] [Q&A] [answered]
<img width="319" alt="image" src="https://user-images.githubusercontent.com/118147452/229721730-324202d8-b3be-41ec-b28f-2d8b39cd29f2.png">
Hello, may I ask how the connecting line can connect two notes together? The connecting line on the score is too short now,

**Comment by Danielku15** [comment, marked as answer]
This is by design. Those are "slide up" which are on purpose not connected to the next note. They are rather played in the way that you slide the hand up the fretboard a few frets. To have the notes connected they need to be written either as legato or shift slides. 

In Guitar Pro:
![image](https://user-images.githubusercontent.com/674916/230610935-2d85e1d5-b6fc-49a8-9746-60dfad4fd0aa.png)

List of slides from one of our tests on how they look:
https://github.com/CoderLine/alphaTab/blob/develop/test-data/visual-tests/notation-legend/slides-default.png
